robottelo/ui/navigator.py

```diff
@@ -20,7 +21,14 @@ def menu_click(self, top_menu_locator, sub_menu_locator,
                 "arguments[0].click();",
                 tertiary_element,
             )
-        self.browser.refresh()
+        # 1328627 - as the number of subs increases, dashboard page gets slower
```

Comment:
yeah, i was just trying to find out the exact point where this has been introduced :)